packages/web-components/fast-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from "@microsoft/fast-element";
+import { forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
+import {
+    accentFillActiveBehavior,
+    accentFillHoverBehavior,
+    accentFillRestBehavior,
+    neutralFillHoverBehavior,
+    neutralFillInputActiveBehavior,
+    neutralFillInputHoverBehavior,
+    neutralFillInputRestBehavior,
+    neutralFillRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+    neutralOutlineRestBehavior,
+} from "../styles/index";
+
+export const TooltipStyles = css`
+    :host {
+        contain: layout;
+        overflow: visible;
+        height: 0;
+        width: 0;
+    }
+
+    .tooltip {
+        box-sizing: border-box;
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--outline-width) * 1px) solid ${accentFillActiveBehavior.var};
+        border-color: ${neutralFocusBehavior.var};
```

Comment:
Can we drop the accent fill behavior from line 28, assuming line 29 is overriding it.